101-streamanalytics-create/azuredeploy.parameters.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "streamAnalyticsJobName": {
+      "value": "StreamAnalyticsJobName"
```

Comment:
ok - unique names tend to be one of the most common failures but if it's in the scope of the subscription that should be easier to spot...